tools/add_triage_signature.py

```diff
@@ -1394,7 +1394,13 @@ def __init__(self, *args, **kwargs):
     def _process_ticket(self, url, issue_key):
         md = get_metadata_json(url)
         cluster_id = md["cluster"]["id"]
-        if md["cluster"]["logs_info"] in ("timeout", "completed"):
+        # temporary workaround till requested status is set only when the controller is
+        # already up and running
+        cluster = md["cluster"]
```

Comment:
why not to check finalizing state? till then must-gather will not be available

Reply:
We need to build on must-gather only if 3 masters are installed till then i don't think it will work.

Reply:
@tsorya what does it mean 3 masters are installed? All in Done ?

Reply:
Yep that means  == finilizing state in cluster

Reply:
But @tsorya we should be able to get must-gather logs if 1 master is done, no?